File: PengembanganAplikasiMobile/2021-2022--sem2/PAM_LayoutActivityIntent/app/src/main/java/com/septovan/pam/layoutactivityintent/HomeActivity.java
package com.septovan.pam.layoutactivityintent;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class HomeActivity extends AppCompatActivity {

    private TextView txtHello;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);

        txtHello = findViewById(R.id.txtHello);

        String user;
        Intent intent = getIntent();
        user = intent.getStringExtra("user");

        txtHello.setText(
                txtHello.getText().toString()
                        .replace("[NAME]", user)
        );
    }
}

File: PengembanganAplikasiMobile/2022-2023--sem2/PAM_AcitivityIntent/app/src/main/java/edu/uksw/fti/pam/acitivityintent/HomeActivity.kt
package edu.uksw.fti.pam.acitivityintent

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material.MaterialTheme
import androidx.compose.material.Surface
import androidx.compose.ui.Modifier
import edu.uksw.fti.pam.acitivityintent.ui.screens.BottomNavigationMainScreenView
import edu.uksw.fti.pam.acitivityintent.ui.theme.PAMAcitivityIntentTheme

class HomeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            PAMAcitivityIntentTheme {
                // A surface container using the 'background' color from the theme
                Surface(
                    modifier = Modifier.fillMaxSize(),
                    color = MaterialTheme.colors.background
                ) {
                    val username = getIntent().getStringExtra("username") ?: ""
                    BottomNavigationMainScreenView()
                    // HomeScreen(username)
                }
            }
        }
    }
}